visualization/app/codeCharta/state/effects/focusNodes/focus.effect.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { TestBed } from "@angular/core/testing"
+import { BehaviorSubject } from "rxjs"
+import { ThreeMapControlsService } from "../../../ui/codeMap/threeViewer/threeMapControls.service"
+import { focusNode, unfocusAllNodes, unfocusNode } from "../../store/dynamicSettings/focusedNodePath/focusedNodePath.actions"
+import { FocusEffects } from "./focus.effect"
+import { MockStore, provideMockStore } from "@ngrx/store/testing"
+import { provideMockActions } from "@ngrx/effects/testing"
+import { CcState, DynamicSettings } from "../../../codeCharta.model"
+import { currentFocusedNodePathSelector } from "../../store/dynamicSettings/focusedNodePath/currentFocused.selector"
+import { focusedNodePathSelector } from "../../store/dynamicSettings/focusedNodePath/focusedNodePath.selector"
+import { isChildPath } from "../../../util/isChildPath"
+import { EffectsModule } from "@ngrx/effects"
+import { Action } from "@ngrx/store"
+
+jest.mock("../../../util/isChildPath", () => ({
+    isChildPath: jest.fn()
+}))
+const mockIsChildPath = isChildPath as jest.Mock
+
+describe("FocusEffects", () => {
+    let actions$: BehaviorSubject<Action>
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    let effects: FocusEffects
```

Comment:
this variable is unused, can it be removed? (even if not, the comment above it should be)